tests/pipeline/examples/pipelines/valid_pipeline/example_1/fourth_component/fondant_component.yaml

```diff
@@ -0,0 +1,28 @@
+name: Fourth component
+description: This is an example component
+image: example_component:latest
+
+consumes:
+  images_data:
+    type: binary
+
+  captions_data:
+    type: string
+
+  embeddings_data:
+    type: array
+    items:
+      type: float32
+
+produces_data:
+  type: binary
+
+args:
+  storage_args:
+    description: Storage arguments
+    type: str
+  some_list:
+    description: Some list
+    type: list
+    items:
+      type: int
```

Comment:
I don't think we can define arguments like this. This is only the case for the fields